premis_event_service/views.py

```diff
@@ -597,26 +624,54 @@ def app_agent(request, identifier=None):
                 page = request.GET.get('page')
             else:
                 page = 1
-            atomFeed = makeObjectFeed(
-                paginator=Paginator(Agent.objects.all(), 20),
-                objectToXMLFunction=objectToAgentXML,
-                feedId=requestString[1:],
-                webRoot="http://%s" % request.META.get('HTTP_HOST'),
-                title="Agent Entry Feed",
-                idAttr="agent_identifier",
-                nameAttr="agent_name",
-                request=request,
-                page=page,
-            )
+            try:
+                atomFeed = makeObjectFeed(
+                    paginator=Paginator(Agent.objects.all(), 20),
+                    objectToXMLFunction=objectToAgentXML,
+                    feedId=requestString[1:],
+                    webRoot="http://%s" % request.META.get('HTTP_HOST'),
```

Comment:
I'm all in favor of the single quotes. it's the double quotes that I think we tend to try to stay away from except for docstrings or strings with literal single quotes in them. but it's not a mandatory change, especially in this repo. single and double quotes are already interspersed throughout.